packages/snap-preact/src/Templates/Stores/TemplateStore.ts

```diff
@@ -66,11 +68,32 @@ export type TemplateStoreComponentConfig = {
 };
 
 export type CommonPlugins = {
-	backgroundFilters?: CommonPluginBackgroundFilterConfig;
-	scrollToTop?: CommonPluginScrollToTopConfig;
+	backgroundFilters?: CommonPluginBackgroundFilterConfig & AbstractPluginConfig;
+	scrollToTop?: CommonPluginScrollToTopConfig & AbstractPluginConfig;
```

Comment:
This shouldn't need `AbstractPluginConfig` since the type definition already has this.